MibTeX/src/de/mibtex/export/ExportFindPDFs.java

```diff
@@ -0,0 +1,91 @@
+package de.mibtex.export;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.function.BiFunction;
+
+import javax.swing.JFileChooser;
+import javax.swing.filechooser.FileNameExtensionFilter;
+
+import de.mibtex.BibtexEntry;
+import de.mibtex.BibtexViewer;
+
+public class ExportFindPDFs extends Export {
+
+	public ExportFindPDFs(String path, String file) throws Exception {
+		super(path, file);
+	}
+
+	@Override
+	public void writeDocument() {
```

Comment:
Split method and fix indentation with auto formatter

Reply:
done